models/user.go

```diff
@@ -1083,6 +1087,7 @@ func VerifyActiveEmailCode(code, email string) *EmailAddress {
 
 // ChangeUserName changes all corresponding setting from old user name to new one.
 func ChangeUserName(u *User, newUserName string) (err error) {
+	oldUserName := u.Name
```

Comment:
Unnecessary local variable?

Reply:
Not unnecessary, this is used after the username is changed, to create the redirects.  Modeled after https://github.com/go-gitea/gitea/blob/master/models/repo.go#L1344

Reply:
I find it more redable too ... I'll mark it as resolved - if any complains pleace ping me :)